network-api/networkapi/templates/pages/primary_nav.html

```diff
@@ -0,0 +1,66 @@
+{% if page.zen_nav == True %}<div id="primary-nav-container" data-nav-mode="zen" class="zen-mode">
```

Comment:
I think I lost some changes in here during the rebase, I need to re copy it to ensure it's not regressing anything.

Reply:
Have you checked this?

Reply:
yup, the conflict was due to internet health links updates. Both links should go to "/internet-health"